tests/test_resources.py

```diff
@@ -218,3 +222,67 @@ async def test_absent_resource(joint_fixture: JointFixture):
         await query_handler.handle_query(
             class_name="does_not_exist", query="", filters=[]
         )
+
+
+@pytest.mark.asyncio
+async def test_resource_deletion(joint_fixture: JointFixture):
+    """Make sure we can delete a resource.
+
+    Verify that the targeted resource is deleted and nothing else.
+    """
+
+    query_handler = await joint_fixture.container.query_handler()
+    all_resources = await query_handler.handle_query(
+        class_name="DatasetEmbedded", query="", filters=[]
+    )
+
+    assert all_resources.count > 0
```

Comment:
```suggestion
    assert all_resources.count > 1
```

Make sure that there is more than one resource (at least one that is not deleted) in the test data.

Reply:
Changed